### PR TITLE
bug fix and tests

### PR DESCRIPTION
**Pull request by JohnStuartBarrie**
# Description

When loading the GAC Gateway pointing at local Master branch of Carbon.

Click on Contacts and Create and New Customer.

On the New Customer Screen open the dropdown for either Account Default or Country

Expected

The dropdown to be directly underneath the button

Observed

It appears off to the side of the page. See attachment.

# Testing Instructions

### CODE REVIEW

**Comment by iliaoni** [comment]
Tested on FF/Chrome/Safari - dropdowns open in correct position. QA passed.

File: src/components/dropdown/dropdown.js [hunk 1]
```diff
@@ -678,13 +677,15 @@ const Dropdown = Input(InputIcon(InputLabel(InputValidation(class Dropdown exten
...
     }
 
-    content.push(
-      <Portal onReposition={ this.calculatePosition }>
-        <div { ...this.listBlockProps } ref={ (node) => { this.listBlock = node; } }>
-          { this.listHTML }
-        </div>
-      </Portal>
-    );
+    if (this.state.open) {
```
Comment (andrewjtait) [inline src/components/dropdown/dropdown.js]: unfortunately we cannot do this due to https://jira-sage.valiantyscloud.net/browse/CARBON-1044

Comment (JohnStuartBarrie) [reply src/components/dropdown/dropdown.js]: This branch was approved tested and merged. Should we revert. This being said, when I read the other ticket, it seems that CARBON-1044 is resolved by this fix right?

Comment (andrewjtait) [reply src/components/dropdown/dropdown.js]: The issue is that automation expects the dropdown list to be present in the DOM, even when closed. Now that you have removed the list from the DOM, automation will fail.

The ticket I linked to needs a solution that will resolve the issue with automation as well.

**Pull request merged**